domain/src/test/java/org/oppia/android/domain/classify/rules/textinput/TextInputContainsRuleClassifierProviderTest.kt

```diff
@@ -24,6 +24,27 @@ import kotlin.test.fail
 @Config(manifest = Config.NONE)
 class TextInputContainsRuleClassifierProviderTest {
 
+  private val STRING_VALUE_CONTAINS_ANSWER_TEXT =
```

Comment:
You had done the correct naming in file `domain/src/test/java/org/oppia/android/domain/classify/rules/textinput/TextInputFuzzyEqualsRuleClassifierProviderTest.kt` 
We should do the same here.

Reply:
Done